packages/preact-query-persist-client/src/__tests__/testPersistProvider.tsx

```
@@ -0,0 +1,61 @@
+/** @jsxRuntime automatic */
+/** @jsxImportSource preact */
+import {
+  persistQueryClientRestore,
+  persistQueryClientSubscribe,
+} from '../../../query-persist-client-core/src'
+import {
+  IsRestoringProvider,
+  QueryClientProvider,
+} from '../../../preact-query/src'
+import type { ComponentChildren } from 'preact'
+import { useEffect, useRef, useState } from 'preact/hooks'
```

ESLint

[error] `preact/hooks` import should occur before import of `../../../query-persist-client-core/src`

(import/order)